ios/swift/AmpliSwiftSampleApp/Shared/ContentView.swift

```diff
@@ -0,0 +1,124 @@
+//
+//  ContentView.swift
+//  Shared
+//
+//  Created by Qingzhuo Zhen on 11/12/21.
+//
+
+import SwiftUI
+import CoreData
+
+struct ContentView: View {
```

Comment:
Can you please add a second view that also uses Ampli to show how the instance is used across the app. e.g. I made this service in node - https://github.com/amplitude/ampli-examples/tree/main/node/typescript/src/services

Reply:
added a TextView and moved loading there